src/main/webapp/app/shared/organization-selector/organization-selector.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import { OrganizationManagementService } from 'app/admin/organization-management/organization-management.service';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { TranslateService } from '@ngx-translate/core';
+import { JhiAlertService } from 'ng-jhipster';
+import { Organization } from 'app/entities/organization.model';
+import { Course } from 'app/entities/course.model';
+
+@Component({
+    selector: 'jhi-organization-selector',
+    templateUrl: './organization-selector.component.html',
+})
+export class OrganizationSelectorComponent implements OnInit {
+    organizations: Organization[];
+    availableOrganizations: Organization[];
+    course: Course;
+
+    constructor(
+        private activeModal: NgbActiveModal,
+        private translateService: TranslateService,
+        private jhiAlertService: JhiAlertService,
+        private organizationService: OrganizationManagementService,
+    ) {}
+
+    ngOnInit(): void {
+        this.organizationService.getOrganizations().subscribe((data) => {
+            this.availableOrganizations = data;
+            if (this.organizations !== undefined) {
+                this.availableOrganizations.forEach((organization) => {
+                    this.organizations.forEach((currentOrganization) => {
+                        if (organization.id === currentOrganization.id) {
+                            this.availableOrganizations = this.availableOrganizations.filter((org) => org.id !== organization.id);
+                        }
+                    });
+                });
```

Comment:
You can use filter and some for this
Like array.filter((item) => fields.some((field) => item[field] === value));

Reply:
Thanks, did not know the 'some' function before :slightly_smiling_face: 
Improved with c5025f2 :+1: